resources/guidelines/testing/store/quality-guidelines-plugins/index.md

```diff
@@ -424,8 +424,8 @@ The project is driven by the *Friends of Shopware* group. You can contribute at
 
 ### Helpful tools for app developers
 
-* Link: [FroshPluginUploader](https://github.com/FriendsOfShopware/FroshPluginUploader) Tool for validating and uploading new SW6 app releases to the Community Store (GitHub Project from "Friends of Shopware").
-* Link: [Shopware CLI tools / console helpers](https://github.com/shopwareLabs/sw-cli-tools)
+* Link: [`shopware-cli`](https://github.com/shopware/shopware-cli) Tool for validating and uploading new Shopware 6 app releases to the Community Store.
+* Link: [`sw-cli-tools` / console helpers](https://github.com/shopwareLabs/sw-cli-tools)
```

Comment:
this is not really maintained anymore

Reply:
I removed it from the "list" and adjusted the description accordingly.